src/Middleware/WebSockets/src/HandshakeHelpers.cs

```diff
@@ -113,8 +115,17 @@ public static string CreateResponseKey(string requestKey)
             using (var algorithm = SHA1.Create())
             {
                 string merged = requestKey + "258EAFA5-E914-47DA-95CA-C5AB0DC85B11";
```

Comment:
I wonder, is it safe to call `Encoding.UTF8.GetBytes` on the base64 string separately from the websocket string and combine them? If so we can avoid this string alloc.